src/Hedgehog/Script.fsx

```diff
@@ -35,7 +36,7 @@ Property.print <| property {
 //
 
 Property.print <| property {
-    let! xs = Gen.list Gen.int
+    let! xs = Gen.list <| Gen.bounded ()
```

Comment:
As mentioned above, I think we should delete `Gen.bounded` now we have the `Range` combinators, so this should be:
```fs
let! xs = Gen.list <| Gen.int <| Range.constantBounded ()
```